### DISCUSSION TITLE
Menu inside table

### DISCUSSION THREAD

**Discussion by remy-poirier** [tailwindlabs/headlessui#1620] [Help]
Hello, first I apologize if this question has already an answer, but I cannot find it.

Here is my problem, as said I have a table where I want to insert a Menu in order to toggle status of my row. But i have a problem, as the Menu goes inside `td` node, it is hidden by the table and other divs above.

I tried a lot of things in order to make it work, but without luck:

- Try to play with `z-index`, `overflows` etc...
- Tried to use Portal, but it shows items far away from table, and I cannot find a solution to make it positioned right under my clicked item

<img width="228" alt="image" src="https://user-images.githubusercontent.com/5598446/175787636-824002fa-d41b-48ce-a5ac-ee235d9ae75b.png">

Do you have guys any solution for this ?

**Comment by bzbetty** [comment]
Generally i'd do either position: fixed or using a portal, then grabbing the mouse co-ordinates from the initiating event to set the top/left.
